cli/framework.py

```diff
@@ -3,12 +3,19 @@
 from __future__ import print_function
 import os
 import sys
+from cli.settings import Settings
 from abc import ABCMeta, abstractmethod
 
+settings = Settings()
 
 class Framework(object):
     __metaclass__ = ABCMeta
 
+    def fetchUserPass(self, env):
+        self.user = settings.getUser()
+        self.passw = settings.getPass(env)
```

Comment:
Can we add a check like "if self.user is None" to make use that user attribute is present , or something like if hasattr(self, user)